app/db/queries/fund.py

```diff
@@ -14,13 +14,13 @@ def add_organisation(organisation: Organisation) -> Organisation:
 def add_fund(fund: Fund) -> Fund:
     db.session.add(fund)
     db.session.commit()
-    current_app.logger.info(f"Fund added with fund_id: '{fund.fund_id}.")
+    current_app.logger.info("Fund added with fund_id: '%s'.", fund.fund_id)
```

Comment:
We might head this way in the future, but for now the consistent approach used everywhere else is:

`current_app.logger.info("Fund added with fund_id: '{fund_id}'.", extra=dict(fund_id=fund.fund_id))`

Reply:
Noted, will convert to this now

Reply:
Is `current_app.logger.info("Fund added with fund_id: '{fund_id}'.", extra={"fund_id": fund.fund_id})` okay or is `dict` syntax preferred?

Reply:
Ah ignore me it's very minor, I'll use `dict` for consistency

Reply:
either/or is totally fine but yeah, they most use `dict()`